update_sst/update_SST_netcdf_fv3.f90

```diff
@@ -0,0 +1,420 @@
+subroutine update_SST_netcdf_fv3 (sstRR, glat, glon, nlon, nlat, xland, vegtyp, ilake,iice,iyear,imonth,iday,ihour)
```

Comment:
Did you use the cold start surface file from chgres-cube?

Because we are doing surface cycling, the surface file should come from restart files.
Please check restart surface files and check tsea, tsfc, and tsfcl.

Reply:
Let me know if you need to discuss this one. 
I also see you comment out some lake functions. Let double check this part of the code also.

Reply:
I double checked the restart sfc_data.nc and found more variables on skin temperature: tsea, tsfc, tsfcl,tisfc, tref. All very similar. So probably update them all? And yes a discussion would help.